gradle.properties

```diff
@@ -1,13 +1,17 @@
 meritVersion=2.11-SNAPSHOT
-#gorm.version=7.0.8.RELEASE
-grailsVersion=4.1.2
-gormVersion=7.0.2
+grailsVersion=5.3.0
```

Comment:
5.3.2 is the latest version, should use that.

Reply:
<img width="1555" alt="Screen Shot 2023-02-24 at 4 01 48 pm" src="https://user-images.githubusercontent.com/90952854/221444396-152e5a12-718f-465b-8d0f-fb017095bc94.png">

The reason why I opted for 5.3.0 is because it cannot download 5.3.2.

Reply:
You need to separate out the variable for the grails-gradle-plugin from the main grails version.  They mostly change together but not always.  See ecodata if you want an example.